src/igvfd/searches/configs/auxiliary_set.py

```diff
@@ -0,0 +1,47 @@
+from snovault.elasticsearch.searches.configs import search_config
+
+
+@search_config(
+    name='AuxiliarySet'
+)
+def cell_line():
```

Comment:
I don't think it is cell_line

Reply:
Sorry about this obvious copy-paste error, fixed